.github/workflows/sdk-integration.yml

```diff
@@ -0,0 +1,82 @@
+name: SDK Integration
+on:
+  pull_request:
+    types:
+    - opened
+    - labeled
+    - synchronize
+jobs:
+  update-sdk-prs:
+    runs-on: ubuntu-latest
+    name: Update SDK PRs
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'public-api') }}
```

Comment:
I would suggest a more descriptive label like "update-SDKs" or similar?

Reply:
The intention here is to use this flow primarily for the public API effort and we'll decide if we want to keep it after the project is complete.